docs/3____physical_signal_abstraction.adoc

```diff
@@ -13,44 +16,37 @@ This section explains the different variable types used to carry network traffic
 To define the signal abstraction layer (or "high cut"), signal variables are defined.
 
 A signal variable carries the physical value of a network signal normally packaged inside a PDU or frame.
-The unit definition of the variable must match the one defined in the <<NetworkDescriptionFile>>.
+The unit definition of the variable must match the one defined in the <<NetworkDescriptionFile>>, if provided.
 
 Each network signal must be listed as <<TerminalMemberVariable>> of its corresponding <<PDUTerminal>>.
 
 In case multiplexed signals are present in a frame/PDU: all signals are present, but only the active signal according to the multiplex switch signal contains a valid value, all inactive variables have undefined values _[those values could even be outside their specified min-max range without fault]_.
 
 Generally, signal variables are optional, but they must either be all missing or all present to clearly indicate support for the signal abstraction layer.
 
-All signal variables are clocked to indicate when exactly they have been sent, see <<ClockVariableHighCut>>.
+All signal variables are clocked to exactly indicate when they have been sent, see <<ClockVariableHighCut>>.
 
 
 ==== Clock Variables [[ClockVariableHighCut]]
```

Comment:
I think we should write all entry points with the same syntax. You can also see the links by clicking on chapters in the table of contents. Do we want to agree on a syntax in this comment and I create an issue @andreas-junghanns? Then I can straighten that out for the entire document as soon as we've gone through the content?